packages/thumbprint-react/components/Image/get-scroll-parent.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import getScrollParent from './get-scroll-parent';
+
+test('defaults to `document.body`', () => {
+    const wrapper = mount(<div>goose</div>);
+    expect(getScrollParent(wrapper.getDOMNode())).toBe(document.body);
+});
+
+test('looks for parents with `overflow: scroll`', () => {
+    const wrapper = mount(
+        <div>
+            <div style={{ overflow: 'scroll' }} data-test-id="parent">
```

Comment:
Is `data-test-id` an internal TUI convention that happens to differ from the public `data-test` API I added to a bunch of TUI components?

Reply:
Nope. That was by accident. Fixed!